src/context.ts

```diff
@@ -9,8 +10,13 @@ const yaml = require('js-yaml')
  * @property {payload} payload - The webhook event payload
  * @property {logger} log - A logger
  */
-class Context {
-  constructor (event, github, log) {
+export class Context {
+  id?: string
```

Comment:
Should ID ever be absent in real-world usage?

Reply:
Was this removed? I get this error if I remove the`?` it because it's not defined anywhere else:

`Property 'id' has no initializer and is not definitely assigned in the constructor.`

I noticed it's not present in `context.js` https://github.com/probot/probot/blob/a0f84205f87cfebd15aa863b8f6cd84bda661f86/lib/context.js#L12-L16

Reply:
I would think not. Can we then define it in the constructor instead? TS complains when it's not optional: `Property 'id' has no initializer and is not definitely assigned in the constructor.`

So we could do:
```js
constructor (event:any, github:GitHubApi, log:LoggerWithTarget) {
    Object.assign(this, event)
    this.github = github
    this.log = log
    this.id = event.id
  }
```